src/concrete/ml/quantization/post_training.py

```diff
@@ -387,16 +384,8 @@ def _calibrate_layers_activation(
             " must return an instance of RawOpOutput. \n"
             f" ** Offending Op: {quantized_op.__class__.__name__}",
         )
-        # For PTQ, the calibration is performed on quantized data. But
-        # raw operation output (RawOpOutput) data should not be quantized
-        if calibrate_quantized and not isinstance(quant_result, RawOpOutput):
-            assert isinstance(quant_result, QuantizedArray)
-            return (
-                quant_result.dequant(),
-                quant_result.quantizer if isinstance(quant_result, QuantizedArray) else None,
-            )
 
-        # For QAT, the calibration is performed on raw data, performing
+        # For both PTQ and QAT, the calibration is performed on raw data, performing
```

Comment:
Can you link to an issue here. We have to track why we did this change somewhere.

Reply:
reverted